infrastructure/clients/meet.zinc.coop/infrastructure.tf

```diff
@@ -50,6 +43,17 @@ resource "aws_instance" "convene_video" {
   key_name = "deployer-key"
 }
 
+data "aws_ami" "convene_ami" {
+  most_recent      = true
+  # name_regex       = "^convene-jitsi-*"
```

Comment:
Do we intend to remove this line or this is useful?

Reply:
Hmm, good quesiton. I think we can delete the line completely; but I _think_ this is what is sent to AWS to filter down the list of images that are returned. We probably want to use this in place of the `filter` block below; since our selection needs are relatively limited.  But I could be [reading the docs wrong](https://www.terraform.io/docs/providers/aws/d/ami.html#name_regex)

Reply:
Nope I have it backwards, we can delete this and rely on the `filter` block.